public/app/plugins/datasource/loki/datasource.ts

```diff
@@ -200,6 +203,7 @@ export class LokiDatasource
     const queryLimit = isMetricsQuery(target.expr) ? options.maxDataPoints : target.maxLines;
     const query = {
       query: target.expr,
+      direction: target.direction,
```

Comment:
We should probably have the same logic here as in range query
`let direction = target.direction ?? DEFAULT_QUERY_PARAMS.direction`

Reply:
Yes, this seems like a good idea.